journal/modernized/21-chapter-18.adoc

```diff
@@ -8,6 +8,7 @@
 * Illness of Daniel Wheeler
 * Sail for Botany Bay
 * Arrival at Sydney Interview with the Governor
+//Line 10 was underlined in book. Should 'Arrival at Sydney' and 'Interview with the govenor' be split into two lines?
```

Comment:
Yes, it looks like these should be split:

![2019-03-01_08-37-58](https://user-images.githubusercontent.com/7050938/53641575-6e9c1c00-3bfd-11e9-8010-c641bd56c41c.png)

Reply:
Oh thanks, I should have looked here! I think i just checked in the original on asciidoc. Sorry!